CONTRIBUTING.md

```diff
@@ -124,8 +124,6 @@ def pt_adjust(pkz:sd, dp1: sd, q_con: sd, pt: sd):
     with computation(PARALLEL), interval(...):
 ```
 
-Note that `fv3core.gtstencil` can be manually called on an undecorated stencil, but this is currently in general discouraged.
```

Comment:
good call, we don't want to even mention this option any more

Reply:
The option has been removed by this PR, so I'd hope so!